watcher.go

```diff
@@ -110,9 +112,16 @@ func (w *Watcher) Start(stop <-chan struct{}) {
 				staticConfigurations = append(staticConfigurations[:toDelete], staticConfigurations[toDelete+1:]...)
 			default: // add or update
 				log.Printf("handle update workload %s", wle.Spec.Address)
-				newTarget := make(map[string][]string)
-				newTarget["targets"] = append(newTarget["targets"], fmt.Sprintf("%s:15020", wle.Spec.Address))
-				staticConfigurations = append(staticConfigurations, newTarget)
+				newTargetAddr := fmt.Sprintf("%s:15020", wle.Spec.Address)
+
+				// Remove duplicates from the node IPs.
+				if checkDuplicates(staticConfigurations, newTargetAddr) {
+					log.Printf("workload target endpoint %s exists in configuration", newTargetAddr)
```

Comment:
what is the frequency here?  should we be concerned at all with the logs volume here?  maybe import the istio logging package and use `Debugf` ?

i'd also just break here and save on the `else` indent that follows.

Reply:
current log is like the code block when initialize the configmap. Any query after initialization won't generate more log.

```
18:04:33 Wed May 19 linggg@linggg: ~
 $ ki logs prometheus-5485f55b86-ksfch -c vm-discovery
2021/05/20 01:02:18 starting new watcher for workload entries
2021/05/20 01:02:18 workload entry watcher created
2021/05/20 01:02:18 waiting to be stopped
2021/05/20 01:02:18 handle update workload 21.0.126.129
2021/05/20 01:02:18 workload 21.0.126.129 already registered as 21.0.126.129:15020
2021/05/20 01:02:18 handle update workload 21.0.125.130
2021/05/20 01:02:18 workload 21.0.125.130 already registered as 21.0.125.130:15020
18:04:38 Wed May 19 linggg@linggg: ~
```